gammapy/spectrum/models.py

```diff
@@ -1464,3 +1465,130 @@ def evaluate(self, energy, **kwargs):
         flux = self.spectral_model.evaluate(energy=energy, **kwargs)
         absorption = self.absorption.evaluate(energy=energy, parameter=parameter)
         return flux * absorption
+
+
+class NaimaModel(SpectralModel):
+    r"""A wrapper for `Naima <https://naima.readthedocs.io/en/latest/>`_ models
+
+    This class provides an interface with the models defined in the `~naima.models` module.
+    The model accepts as a positional argument a Naima
+    `radiative model <https://naima.readthedocs.io/en/latest/radiative.html>`_ instance, used
```

Comment:
If you just put this, does it create a link to the Naima docs?
```
`naima`_
```
I think it should because of https://github.com/gammapy/gammapy/blob/7b186a76debf91611dd6e52651a33b94e7cf9e37/docs/conf.py#L74

Please run `python setup.py build_docs` locally and check the HTML output.